### PR TITLE
Enable Local Storage for Word List

### PR DESCRIPTION
**Pull request by Jonathan75**

### CODE REVIEW

File: mockup-in-jquery.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
+<html>
```
Comment (Jonathan75) [inline mockup-in-jquery.html]: You can ignore this mockup file. It was just a proof of concept.

File: src/components/WordList.js [hunk 2]
```diff
@@ -1,15 +1,11 @@
 import React, { Component } from 'react'
 
 class WordList extends Component {
-  constructor(props) {
-    super(props)
-  }
-
   render() {
-    const wordListItems = this.props["wordList"].map((word) => <li>{word}</li>)
+    const wordListItems = this.props.wordList.map((word) => <li>{word}</li>)
```
Comment (brendanthomas1) [inline src/components/WordList.js]: I'd just do this inline on line 8 instead of making a variable for it.

File: src/App.js [hunk 3]
```diff
@@ -6,14 +6,20 @@ class App extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      wordList: []
+      wordList: JSON.parse(localStorage.getItem("wordList") || "[]")
     }
     this.addWord = this.addWord.bind(this);
   }
 
+  storWordList = (wordList) => {
```
Comment (brendanthomas1) [inline src/App.js]: Your spelling test app needs a spell check 😆 😄

File: src/App.js [hunk 4]
```diff
@@ -6,14 +6,20 @@ class App extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      wordList: []
+      wordList: JSON.parse(localStorage.getItem("wordList") || "[]")
```
Comment (brendanthomas1) [inline src/App.js]: 🤔I haven't used local storage with React before, but I'm wondering if this is the best place to do this? React recommends loading data from a remote endpoint in [componentDidMount()](https://reactjs.org/docs/react-component.html#componentdidmount), but I'm not sure if they have an opinion on local storage. Might be worth a Googling if you're interested, but this probably also works.

File: src/App.js (hunk 3 above)
Comment (Jonathan75) [reply src/App.js]: Thanks

File: src/components/WordList.js (hunk 2 above)
Comment (Jonathan75) [reply src/components/WordList.js]: Also need a key.

File: src/App.js (hunk 4 above)
Comment (Jonathan75) [reply src/App.js]: Need to read this more but looked like some good documentation https://reactjs.org/docs/state-and-lifecycle.html

File: src/App.js [hunk 5]
```diff
@@ -11,9 +11,19 @@ class App extends Component {
...
+
+  storeWordList = (wordList) => {
+    localStorage.setItem("wordList", JSON.stringify(wordList))
+  }
+
   addWord = (e) => {
     e.preventDefault()
-    this.setState({wordList: this.state.wordList.concat(e.target.elements["word"].value)})
+    const updatedWordList = this.state.wordList.concat(e.target.elements["word"].value)
+    this.setState({wordList: updatedWordList})
```
Comment (Jonathan75) [inline src/App.js]: Think we can add a function call after the status updates.

**Pull request merged**